frontend/pages/admin/IntegrationsPage/cards/MdmSettings/AndroidMdmPage/AndroidMdmPage.tsx

```diff
@@ -21,17 +27,68 @@ import TurnOffAndroidMdmModal from "./components/TurnOffAndroidMdmModal";
 
 const baseClass = "android-mdm-page";
 
-const TurnOnAndroidMdm = () => {
+const POPUP_WIDTH = 885;
+const POPUP_HEIGHT = 600;
+
+interface ITurnOnAndroidMdmProps {
+  router: InjectedRouter;
+}
+
+const TurnOnAndroidMdm = ({ router }: ITurnOnAndroidMdmProps) => {
   const { renderFlash } = useContext(NotificationContext);
+
+  // TODO: figure out issue with aborting the SSE fetch when the window is closed
+  const newWindow = useRef<Window | null>(null);
+
   const [fetchingSignupUrl, setFetchingSignupUrl] = useState(false);
+  const [setupSse, setSetupSse] = useState(false);
+
+  const handleSSE = useCallback(
+    async (abortController: AbortController) => {
+      try {
+        await mdmAndroidAPI.startSSE(abortController.signal);
+        abortController.abort();
```

Comment:
Just curious, is it recommended to call `abort` on success?

Reply:
ah this should probably be called after either a success or error. will change

Reply:
actually i think its fine here. but I'm not sure about your original question. let me try to remove it and see what happens